Src/AutoFixtureUnitTest/Kernel/SeededFactoryTest.cs

```diff
@@ -129,5 +129,23 @@ public void CreateWithCorrectlyTypedSeededRequestWithNullSeedReturnsCorrectResul
             Assert.Equal(expectedResult, result);
             // Teardown
         }
+
+        [Fact]
+        public void CreateWithCorrectlyTypedRequestReturnsCorrectResult()

```

Comment:
Doesn't this test slightly overspecify the implementation details?

If we didn't have this test, wouldn't it leaves us more flexibility with regards to future refactoring?